From: Jeremy Bicha <jbicha@ubuntu.com>
Date: Tue, 5 May 2020 19:26:45 -0400
Subject: fix-openexr-check

This is a simple workaround since we know our openexr version
doesn't work with C++98 any more. Easier than porting this commit
to autotools.

https://gitlab.freedesktop.org/gstreamer/gst-plugins-bad/-/commit/d3d422897

See also
https://gitlab.freedesktop.org/gstreamer/gst-plugins-bad/-/commit/91e430727
---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index 843c8fd..d60f8a2 100644
--- a/configure.ac
+++ b/configure.ac
@@ -1897,7 +1897,7 @@ translit(dnm, m, l) AM_CONDITIONAL(USE_OPENEXR, true)
 AG_GST_CHECK_FEATURE(OPENEXR, [openexr library], openexr, [
   AG_GST_PKG_CHECK_MODULES(OPENEXR, OpenEXR)
   if test "x$HAVE_OPENEXR" = "xyes"; then
-    AG_GST_PKG_CHECK_MODULES(OPENEXR240, OpenEXR >= 2.4.0)
+    AG_GST_PKG_CHECK_MODULES(OPENEXR240, OpenEXR >= 2.3.0)
     if test "x$HAVE_OPENEXR240" = "xyes"; then
       OPENEXR_STDCFLAGS=
     else
